src/inputs/labeled-inputs.scss

```diff
@@ -23,7 +23,8 @@
   &.iui-inline-icon {
     > .iui-input,
     > .iui-textarea {
-      padding-right: $iui-icons-default + $iui-l;
+      padding-right: ($iui-icons-default + $iui-l);
```

Comment:
So it won't work on IE. Is it worth having this when you started working on this PR https://github.com/iTwin/iTwinUI/pull/365?

Reply:
I am not sure if it's worth it. That's why I didn't merge it

Reply:
#365 is kind of in a bad state and I might rewrite the code in there.

So maybe we should merge this? It can't hurt (IE will still use 40px), and might even help in cases where the borderless button width is not 40px.

Reply:
Before merge, can we come up with a standard to our variables namings? Something like iui-{component}-{element}-{value}

Reply:
So in this case it would be something like `--iui-input-icon-inline-padding`?

Reply:
amm..something, even though it sounds like it is padding for icon, whereas actually it is padding of input itself

Reply:
cant it be just `--iui-input-right-padding` ?
Yes, it is now used only in inlice icon context, but.. maybe it could be used in other cases, just to set right padding dynamically???

Reply:
I looked at React implementation and it is a bit ugly 😕 You need to handle all of the sizing when in reality correct CSS could do that easily.
![image](https://user-images.githubusercontent.com/36186912/141249750-efc01ee3-bf0a-457c-b218-75f12c97437c.png)

Reply:
> You need to handle all of the sizing when in reality correct CSS could do that easily.

what do you mean by "correct CSS"?

Reply:
Having button next to input not inside 😀